integtests/tools/components/test_tools.py

```diff
@@ -197,118 +197,142 @@ async def test_create_config(mcp_context: Context, configs: list[ConfigDef], keb
         )
 
 
-@pytest.mark.asyncio
-async def test_update_config(mcp_context: Context, configs: list[ConfigDef], keboola_project: ProjectDef):
-    """Tests that `update_config` updates a configuration with correct metadata."""
+@pytest.fixture
+def mcp_server(storage_api_url: str, storage_api_token: str, workspace_schema: str) -> FastMCP:
+    config = Config(storage_api_url=storage_api_url, storage_token=storage_api_token, workspace_schema=workspace_schema)
+    return create_server(config)
 
-    # Use the first component from configs for testing
-    test_config = configs[0]
-    component_id = test_config.component_id
 
-    # Define initial configuration test data
-    initial_name = 'Initial Test Configuration'
-    initial_description = 'Initial test configuration created by automated test'
-    initial_parameters = {'initial_param': 'initial_value'}
-    initial_storage = {'input': {'tables': [{'source': 'in.c-bucket.table', 'destination': 'input.csv'}]}}
+@pytest_asyncio.fixture
+async def mcp_client(mcp_server: FastMCP) -> AsyncGenerator[Client, None]:
+    async with Client(mcp_server) as client:
+        yield client
 
-    # Define updated configuration test data
-    updated_name = 'Updated Test Configuration'
-    updated_description = 'Updated test configuration by automated test'
-    updated_parameters = {'updated_param': 'updated_value'}
-    updated_storage = {'output': {'tables': [{'source': 'output.csv', 'destination': 'out.c-bucket.table'}]}}
-    change_description = 'Automated test update'
 
-    client = KeboolaClient.from_state(mcp_context.session.state)
+@pytest_asyncio.fixture
+async def initial_cmpconf(
+    mcp_client: Client, configs: list[ConfigDef], keboola_client: KeboolaClient
+) -> AsyncGenerator[ConfigToolOutput, None]:
+    # Create the initial component configuration test data
+    tool_result = await mcp_client.call_tool(
```

Comment:
Shouldn't the initial config be created by the storage client?

1. It's more straightforward
2. I think in the init phase it's better to avoid using tools which themselves are being tested